packages/toolkit/src/components/ClonePipelineDialog.tsx

```diff
@@ -182,43 +219,157 @@ export const ClonePipelineDialog = ({
+                                                      variant="lightNeutral"
+                                                    >
+                                                      user
+                                                    </Tag>
+                                                  )}
+                                                </span>
+                                              </div>
+                                            </Select.Item>
+                                          )
+                                        )}
+                                    </Select.Group>
+                                  </Select.Content>
+                                </Select.Root>
+                              </Form.Control>
+
+                              <Form.Message />
+                            </Form.Item>
+                          );
+                        }}
+                      />
+
+                      <span className="pt-[30px] text-2xl text-semantic-fg-disabled">
+                        /
+                      </span>
+
+                      <Form.Field
+                        control={form.control}
+                        name="id"
+                        render={({ field }) => {
+                          return (
+                            <Form.Item className="w-full">
+                              <Form.Label className="product-body-text-3-semibold">
+                                Pipeline Name
+                              </Form.Label>
+                              <Form.Control className="p-[7px]">
+                                <Input.Root>
+                                  <Input.Core
+                                    {...field}
+                                    className="pl-2 !product-body-text-2-regular"
+                                    type="text"
+                                    placeholder="Pipeline name"
+                                    required={false}
+                                    value={field.value || ""}
+                                  />
+                                </Input.Root>
+                              </Form.Control>
+
+                              <Form.Message />
+                            </Form.Item>
+                          );
+                        }}
+                      />
+                    </div>
+                    <p className="text-semantic-fg-secondary product-body-text-3-regular">
+                      <span>Your pipeline URL in Instill AI will be:</span>
+                      <span className="ml-2 break-all product-body-text-3-semibold">
+                        {form.getValues("id") !== "" || form.getValues("id")
+                          ? `${env(
+                              "NEXT_PUBLIC_CONSOLE_BASE_URL"
+                            )}/${form.getValues(
+                              "namespaceId"
+                            )}/pipelines/${form.getValues("id")}`
```

Comment:
@iamnamananand996 getValues is not a reactive subscription, so it will only update the value when the whole component re-render. This is not the UX we want. 

We should use form.watch("id") to be reactive to the data